lego/apps/articles/migrations/0012_migrate_authors.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.0.9 on 2023-03-16 13:13
+
+from django.conf import settings
+from django.db import migrations
+
+
+def merge_authors(apps, schema_editor):
+    Article = apps.get_model("articles", "Article")
+    for e in Article.objects.all():
+        if e.created_by != None:
+            e.authors.add(e.created_by)
+            e.save()
```

Comment:
I just realized this doesn't make sense when `authors` is not empty. Might be an idea to check if `authors` is empty as well :) (In theory this should not happen, but it's a bad idea to depend on things going the way you assume)